checkout.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: lesson
+subtitle: Checkout Procedure
+calendar: https://calendar.google.com/calendar/embed?src=oseuuoht0tvjbokgg3noh8c47g%40group.calendar.google.com
+demopad: http://pad.software-carpentry.org/teaching-demos
+discussionpad: http://pad.software-carpentry.org/lesson-discussion-2016
+email: training-help@software-carpentry.org
+---
+After you have finished the two-day training class,
+you must go through these steps in order to complete your training and be certified:
+
+1.  Submit a [small change to one of our lessons](#lesson-change).
+2.  Take part in an online [discussion session](#discussion-session).
+3.  Teach a short [demonstration lesson](#demo-lesson) online.
```

Comment:
We need to be clear that they need to complete steps 2 and 3 for SWC and DC separately(?) if they want to be certified for both. At the moment this is not very clear and the details of (1) below are even more confusing ("Note that you may submit a change to a Data Carpentry lesson when qualifying as a Software Carpentry instructor").